client/client.go

```diff
@@ -327,18 +399,8 @@ func dialOpenSSL(conf Config, network string, addr string) (net.Conn, error) {
 func newHttpsClient(conf Config) (*http.Client, error) {
 	client := newHttpClient()
 
-	trustedcerts := loadServerTrust(&conf)
-
-	if conf.NoVerify {
-		log.Warnf("certificate verification skipped..")
-	}
-	tlsc := tls.Config{
-		RootCAs:            trustedcerts,
-		InsecureSkipVerify: conf.NoVerify,
-	}
 	transport := http.Transport{
-		TLSClientConfig: &tlsc,
```

Comment:
Removed, since I can't see that it is no longer needed, since the `OpenSSL` lib now handles the TLS communication